.github/workflows/canary-release.yml

```diff
@@ -41,5 +39,5 @@ jobs:
           yarn run changeset version --snapshot $GIT_HASH
           yarn run changeset publish --no-git-tag --tag canary
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.githubToken }}
```

Comment:
Looks like we're mostly using upper-cased names [elsewhere](https://github.com/chromaui/chromatic/blob/main/.github/workflows/playwright-tests.yml#L27), should we do the same here?

Reply:
This secret is passed in as an argument from the workflow that kicks this one off. Its inputs are defined up at the top in the `on:` block. So they are different than env vars, which are typically upper-cased, so I like the different naming so they aren't confused with env vars. What do you think?

Reply:
Got it. I'm ok with that.